nerves_defconfig

```diff
@@ -26,13 +26,14 @@ BR2_ROOTFS_OVERLAY="${BR2_EXTERNAL_NERVES_PATH}/board/nerves-common/rootfs_overl
 BR2_ROOTFS_POST_BUILD_SCRIPT="${NERVES_DEFCONFIG_DIR}/post-build.sh ${BR2_EXTERNAL_NERVES_PATH}/board/nerves-common/post-build.sh"
 BR2_ROOTFS_POST_IMAGE_SCRIPT="${NERVES_DEFCONFIG_DIR}/post-createfs.sh"
 BR2_LINUX_KERNEL=y
-BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
-BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="$(call github,raspberrypi,linux,1.20211007)/linux-raspberrypi-kernel_1.20211007.tar.gz"
+BR2_LINUX_KERNEL_CUSTOM_GIT=y
+BR2_LINUX_KERNEL_CUSTOM_REPO_URL="https://github.com/raspberrypi/linux.git"
```

Comment:
This is interesting. Is Linux.git versioned controlled or will we get whatever the latest they publish here?

Reply:
It's locked to git hash bad3872df40dd9a4ba7ff239c17288a7a84a80f9. This is the next line.

Reply:
I went latest and greatest to be able to use the new `bcm2710-rpi-zero-w-w.dts` device tree file. It boots without it, but there weren't many new commits over the latest tag and I didn't want to do the device tree one way and this switch in a few weeks.